src/main/resources/marc4bibframe.yml

```diff
@@ -1705,3 +1640,91 @@ bibSharedRules:
             CODE: 15,18
         constants:
           LINK: http://id.loc.gov/vocabulary/countries/%CODE
+  776_mappings:
+    types: WORK
+    parent: WORK
+    predicate: OTHER_VERSION
```

Comment:
For 775, we have to create "otherEdition" edge between the works. (as described in MODLD-376)
For 776, we have to create "otherVersion" edge. (as described in MODLD-387)

Reply:
Thank you @pkjacob! I didn't notice it.
Fixed